dttools/src/jx_function.c

```diff
@@ -771,12 +771,41 @@ struct jx *jx_function_schema(struct jx *orig_args, struct jx *ctx) {
 		}
 	}
 	jx_delete(args);
-	jx_delete(context);
+	jx_delete(objlist);
 	return result;
 
 	FAILURE:
 	jx_delete(args);
-	jx_delete(context);
+	jx_delete(objlist);
+	FAIL(funcname, orig_args, err);
+}
+
+struct jx *jx_function_like(struct jx *orig_args, struct jx *ctx) {
+	assert(orig_args);
+	assert(jx_istype(ctx, JX_OBJECT));
+	const char *funcname = "like";
+	const char *err = NULL;
+
+	struct jx *args = jx_copy(orig_args);
+	struct jx *val = jx_eval(jx_array_shift(args), ctx);
```

Comment:
You're gonna be so upset....Your latest push double-`eval()`s. The arguments already get evaluated in `jx_eval_operator()`, so there's no need to eval again here

Reply:
I'll remove that and push the change along with the doc changes in a bit.